deltachat-ios/Controller/ChatListController.swift

```diff
@@ -465,6 +467,7 @@ class ChatListController: UITableViewController {
 
     override func setEditing(_ editing: Bool, animated: Bool) {
         super.setEditing(editing, animated: animated)
+        tableView.setEditing(editing, animated: animated)
         viewModel?.setEditing(editing)
     }
```

Comment:
we set the edit-mode for _three_ objects here.

why do we set the edit-mode only for _two_ objects a few lines below in `setLongTapEditing` - but cancel again _three_ objects.

this looks like some imbalance.

sorry for nitpicking, i just try to understand :)

Reply:
thanks! i was just seeing 2 there and 3 there without understanding details :)